### PR TITLE
Update hotkeys.md

### PR DESCRIPTION
**Pull request by suearkinunity**
It's not really possible to maintain a shortcuts page, and since the Editor has a shortcut manager, it's not really useful to try. Therefore, just telling people these exists and where to find and edit them.

### Purpose of this PR

Update the keyboard shortcuts page of the documentation. 

### Links

**Jira:** https://jira.unity3d.com/browse/DOCATT-5831

### Comments to Reviewers

Documentation only

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=520) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=520) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=520) it.</sub>

File: Documentation~/hotkeys.md [hunk 1]
```diff
@@ -1,46 +1,5 @@
...
-| **Ctrl/Cmd+K**                                               | Create a [new Mesh cube](Cube.md).                           |
-| **Ctrl/Cmd+Shift+K**                                         | Activate the [Shape tool](shape-tool.md).                    |
-| **P**                                                        | Toggle between the three [orientation modes](HandleAlign.md) of the ProBuilder selection handle. |
-| **Shift+Drag** (while moving, rotating, or scaling)          | Extrude [edges](Edge_Extrude.md) or [faces](Face_Extrude.md). |
-| **Shift+Hover**                                              | Show tooltips when hovering over an action icon in the ProBuilder toolbar. |
-| **0**                                                        | In the [UV Editor](uv-editor.md), resets the canvas by centering the Texture in the UV viewer (the workspace area of the UV Editor). |
+For a list of all shortcuts, and to add shortcuts, go to **Main MEnu** > **Unity** > **Settings**.
```
[the remark this answers is not in the document]
Comment (suearkinunity) [reply Documentation~/hotkeys.md:5]: Mac and Windows have different menu names?

[the remark this answers is not in the document]
Comment (JonMUnity) [reply Documentation~/hotkeys.md:5]: **Edit > Shortcuts** is how you open the Shortcuts manager on my machine. How about something like 

```suggestion
For a list of all shortcuts, and to add shortcuts, go to **Edit > Shortcuts** to open the [Shortcuts manager](xref:ShortcutsManager).
```

**Review by JonMUnity — APPROVED** [review]

**Comment by suearkinunity** [comment]
@modrimkus please could you merge this?

**Pull request merged**